src/components/common/snow-background/index.tsx

```
@@ -0,0 +1,88 @@
+
+      // Clear canvas
+      ctx.clearRect(0, 0, canvas.width, canvas.height)
+
+      // Draw particles
+      particles.forEach((particle) => {
+        ctx.beginPath()
+        ctx.arc(particle.x, particle.y, particle.radius, 0, Math.PI * 2)
+        ctx.fillStyle = 'white'
+        ctx.fill()
+
+        if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)